packages/account/src/Sections/Security/ClosingAccount/__tests__/closing-account-reason-form.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ClosingAccountReasonForm from '../closing-account-reason-form';
+
+describe('<ClosingAccountReasonForm />', () => {
+    const mock_props: React.ComponentProps<typeof ClosingAccountReasonForm> = {
+        onConfirmClick: jest.fn(),
+        onBackClick: jest.fn(),
+    };
+
+    const other_financial_priorities_text = /i have other financial priorities/i;
+
+    it('Should render ClosingAccountReasonForm component', () => {
+        render(<ClosingAccountReasonForm {...mock_props} />);
+        expect(screen.getByLabelText(/i want to stop myself from trading./i)).toBeInTheDocument();
+        expect(
+            screen.getByPlaceholderText(/if you don’t mind sharing, which other trading platforms do you use?/i)
+        ).toBeInTheDocument();
+
+        expect(screen.getByRole('button', { name: /Continue/i })).toBeDisabled();
+    });
+
+    it('should call the onBackClick function when cancel button is clicked', () => {
+        render(<ClosingAccountReasonForm {...mock_props} />);
+
+        userEvent.click(screen.getByRole('button', { name: /Back/i }));
+
+        expect(mock_props.onBackClick).toHaveBeenCalledTimes(1);
+    });
+
+    it('Should be disabled when no reason has been selected', async () => {
+        render(<ClosingAccountReasonForm {...mock_props} />);
+
+        fireEvent.click(screen.getByRole('checkbox', { name: other_financial_priorities_text }));
```

Comment:
`userEvent.click`?
since userEvent is preferred above fireEvent, check whether userEvent.change too can be used.
could you check this in other test files as well?